pronto/api/signature.py

```diff
@@ -647,3 +650,26 @@ def digest(self, obj: dict):
             key = "similarity"
 
         return i, j, -obj["residues"][key], -obj[key]
+
+
+def check_if_ncbifam_amr(method_ac):
+
+    con = utils.connect_oracle()
+    cur = con.cursor()
+    cur.execute(
+        f"""
+        SELECT *
+        FROM INTERPRO.NCBIFAM_AMR
+        WHERE METHOD_AC = '{method_ac}'"""
+    )
+
+    return len(cur.fetchall()) > 0
+
+def ncbifam_amr_err_msg(accession):
+    return {
+        "status": False,
+            "error": {
+                "title": "Can't integrate signature",
+                "message": f"<strong>{accession}</strong> is an NCBIfam AMR model."
+            }
+        }
```

Comment:
- Add an empty line above the function (so functions are separated by two blank lines, which is a PEP-8 recommendation)
- Add an empty line below the function (so the file ends with a new line character, which is also a PEP-8 recommendation)
- Type hinting
- Do not include HTML and API response

```suggestion

def ncbifam_amr_err_msg(accession: str) -> dict:
    return {
        "status": False,
        "error": {
            "title": "Can't integrate signature",
            "message": f"{accession} is an NCBIFAM AMR model."
        }
    }

```